.github/workflows/markdown-links-verifier.yml

```diff
@@ -0,0 +1,14 @@
+name: Markdown links verifier
+on: [push, pull_request]
+
+jobs:
+  validate_links:
+    name: Markdown links verifier
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout the repository
+      uses: actions/checkout@v1
```

Comment:
actions should use `@commits` instead of `@versions`. This helps security issues related to something deleting and recreating a version with malicious code. Same for your action below. I like to use the following format to add a comment of what version the commit maps to `owner/repo@commit #@version` https://github.com/dotnet/docs/blob/main/.github/workflows/build-validation.yml#L39